src/R/Editor/Impl/Signatures/SignatureHelp.cs

```diff
@@ -163,10 +163,13 @@ public partial class SignatureHelp : ISignature {
                 IREditorDocument document = REditorDocument.TryFromTextBuffer(SubjectBuffer);
                 if (document != null) {
                     SnapshotPoint? p = REditorDocument.MapCaretPositionFromView(TextView);
+                    var textView = TextView; // Cache so it doesn't go away when callback is invoked.
```

Comment:
However, in this particular case we may technically skip it since if signature is gone, probably there is no need to recalc parameter position.